src/toil/test/wdl/builtinTest.py

```diff
@@ -230,6 +231,12 @@ def testFn_Transpose(self):
         self.assertEqual([[0]], transpose([[0]]))
         self.assertRaises(AssertionError, transpose, [[0, 1, 2], [3, 4, 5, 6]])
 
+    def testFn_Length(self):
+        """Test the wdl built-in functional equivalent of 'length()'."""
```

Comment:
Isn't it just the "WDL 'length()' built-in"? It's not a functional equivalent of anything else, right?

Reply:
Are you suggesting that I should change it to something like `"""Test the WDL 'length()' built-in"""`? That makes sense but I was following the docstring from the other tests to keep them consistent.

Reply:
Yeah, that would make more sense to me.